src/components/IconButton.tsx

```diff
@@ -31,6 +33,7 @@ export function IconButton(props: IconButtonProps) {
     tooltip,
     menuTriggerProps,
     openInNew,
+    compact = false,
```

Comment:
Should we check the presentation settings, i.e. icons in grid tables auto-compact? Or is that not desirable? I guess probably not...

Reply:
Good question. I don't know of a case right now where the button size is dependent on the presentation settings or where its located. I'd add presentation setting support once we need it, or see it is necessary.

Reply:
Yep, sounds good